src/custom/hooks/useGetGpApiStatus.ts

```diff
@@ -0,0 +1,39 @@
+import ms from 'ms.macro'
+import { useState, useEffect } from 'react'
+
+export type GpQuoteStatus = 'COWSWAP' | 'LEGACY'
+// TODO: use actual API call
```

Comment:
Can you point to an issue in the backend repo, if we want to really give them control of this and change it in the future?

Reply:
https://github.com/gnosis/gp-v2-contracts/issues/904